server/covmanager/serializers.py

```diff
@@ -150,3 +150,25 @@ def create(self, attrs):
 
         # Create our ReportConfiguration instance
         return super(ReportConfigurationSerializer, self).create(attrs)
+
+
+class ReportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Report
+        fields = (
+            'id', 'data_created', 'public', 'coverage', 'is_monthly', 'is_quarterly'
```

Comment:
Is it worth adding a `validate()` method to ensure that we don't get `is_monthly=False` + `is_quarterly=True` ?

Reply:
I double-checked the code and all locations check `is_quarterly` first, none of it actually depends on them being both true. So I think we don't need to validate that. In fact, if that was a required property, I think we shouldn't use two bools in the first place in the model.